pkgs/applications/version-management/git-and-tools/default.nix

```diff
@@ -201,6 +201,7 @@ let
 
   gitui = callPackage ./gitui {
     inherit (darwin.apple_sdk.frameworks) Security AppKit;
+    inherit (pkgs) openssl perl;
```

Comment:
Is this really required? None of the other packages do that.

Reply:
Yes, it is in the `openssl-sys` build script, and I am not trace the who need `openssl-sys`.

Reply:
Here is the dependency tree with `openssl`

>>>
gitui v0.11.0 (...)
├── anyhow v1.0.36
├── asyncgit v0.11.0 (...)
│   ├── git2 v0.13.12
│   │   ├── bitflags v1.2.1
│   │   ├── libc v0.2.81
│   │   ├── libgit2-sys v0.12.14+1.1.0
│   │   │   ├── libc v0.2.81
│   │   │   ├── libssh2-sys v0.2.20
│   │   │   │   └── openssl-sys v0.9.58
│   │   │   │       └── libc v0.2.81
│   │   │   └── openssl-sys v0.9.58 (*)
│   │   ├── openssl-sys v0.9.58 (*)
>>>

Reply:
Here is the error if the `openssl` and `perl` remove
``` 
error: failed to run custom build command for `openssl-sys v0.9.58`

Caused by:
  process didn't exit successfully: `/build/source/target/release/build/openssl-sys-7a548c8246aa8cd3/build-script-main` (exit code: 101)
  --- stdout
  cargo:rustc-cfg=const_fn
  cargo:rerun-if-env-changed=X86_64_UNKNOWN_LINUX_GNU_OPENSSL_NO_VENDOR
  X86_64_UNKNOWN_LINUX_GNU_OPENSSL_NO_VENDOR unset
  cargo:rerun-if-env-changed=OPENSSL_NO_VENDOR
  OPENSSL_NO_VENDOR unset
  CC_x86_64-unknown-linux-gnu = Some("/nix/store/b96dqbx6pri2xp2xxlq6i269virrdaw6-gcc-wrapper-9.3.0/bin/cc")
  CFLAGS_x86_64-unknown-linux-gnu = None
  CFLAGS_x86_64_unknown_linux_gnu = None
  HOST_CFLAGS = None
  CFLAGS = None
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some("false")
  CARGO_CFG_TARGET_FEATURE = Some("fxsr,sse,sse2")
  running "perl" "./Configure" "--prefix=/build/source/target/x86_64-unknown-linux-gnu/release/build/openssl-sys-9a6c36981fda3786/out/openssl-build/install" "no-dso" "no-ssl3" "no-unit-test" "no-comp" "no-zlib" "no-zlib-dynamic" "no-md2" "no-rc5" "no-weak-ssl-ciphers" "no-camellia" "no-idea" "no-seed" "no-shared" "linux-x86_64" "-O2" "-ffunction-sections" "-fdata-sections" "-fPIC" "-m64"

  --- stderr
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', /build/gitui-0.11.0-vendor.tar.gz/openssl-src/src/lib.rs:411:39
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
builder for '/nix/store/s9ld60ajf2pka8azyycsh6myykjs86dk-gitui-0.11.0.drv' failed with exit code 101
error: build of '/nix/store/s9ld60ajf2pka8azyycsh6myykjs86dk-gitui-0.11.0.drv' failed
```

Reply:
I mean why can'T we put openssl and perl as a normal input and inherit it?

Reply:
Thanks for reviewing on this.  I will take care on this when `gitui` go to 0.12.  Thanks.

Reply:
You are sure you will remember this? I am not.

Reply:
I'll remember this. No worry.